platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheIncompatibleTasksIntegrationTest.groovy

```diff
@@ -258,29 +313,6 @@ class ConfigurationCacheIncompatibleTasksIntegrationTest extends AbstractConfigu
         "provider { task.project }.map { it.name }" || _
     }
 
-    def "tasks that access project at execution time emit problems when incompatible task is present"() {
```

Comment:
ℹ️  Replaced by a sibling test `execution-time problems in tasks that are not marked incompatible are treated as interrupting failures when incompatible tasks are also scheduled`